### PR TITLE
added feature for adding contact

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
### Description
Created feature file for adding new contact with two scenario outline : adding contact with full details and adding contact without full details.
### Realted Issue : https://github.com/SagarGi/FeatureFiles/issues/4

### CODE REVIEW

File: Features3/tests/features/addContact.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Feature: Add new contact
+    As a user
+    I want to add contact 
+    So that it will be displayed in contact list
+
+    Background:
+        Given User is logged in and navigated to Contact Fox page
```
Comment (SagarGi) [inline Features3/tests/features/addContact.feature]: make past tense

File: Features3/tests/features/addContact.feature [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be created and displayed in contact list
+        Examples:
+            |name|email        |phone     |contact-type|
+            |abc |abc@gmail.com|9801234567|personal    |
+            |xyz |xyz@gmail.com|9874561230|professional|
+
+    Scenario Outline: Add contact without full details
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should not be created
```
Comment (SagarGi) [inline Features3/tests/features/addContact.feature]: i think with email and name we can create a contact (no need of phone number) pleas check once manaully

File: Features3/tests/features/addContact.feature [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        When User enters name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be created 
+        And contact with name as "<name>", email as "<email>", phone as "<phone>" and contact type as "<contact-type>" should be displayed in contact list
+        Examples:
+            |name|email|phone|contact-type|
+            |abc|abc@gmail.com|9801234567|personal|
+            |xyz|xyz@gmail.com|9874561230|professional|
+            |abc|abc@gmail.com||personal|
+            |xyz|xyz@gmail.com||professional|
+
```
Comment (kiranadh1452) [inline Features3/tests/features/addContact.feature]: Please format this part for readability.

Comment (kiranadh1452) [inline Features3/tests/features/addContact.feature]: Please format this part for readability.

File: Features3/tests/features/addContact.feature [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        When User enters name as "<name>", email as "<email>" and contact type as "<contact-type>"
+        Then contact with name as "<name>", email as "<email>" and contact type as "<contact-type>" should not be created 
+        Examples:
+            |name|email|contact-type|
+            ||abc@gmail.com|personal|
+            ||abc@gmail.com|professional|
+            |abc||personal|
+            |abc||professional|
+            |||personal|
+            |||professional|
```
Comment (kiranadh1452) [inline Features3/tests/features/addContact.feature]: Same with this one as well. Formatting would make it easier the table data.

File: Features3/tests/features/addContact.feature [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    Scenario: Adding contact with incorrect email format
+        When User adds contact with following details
+            |name|email|phone|contact-type|
+            |abc|abcgmail.com|9801234567|personal|
+            |abc|abcgmail.com|9801234567|professional|
+        Then validation error message as "Please include an '@' in the email address." should pop up in the input field
```
Comment (kiranadh1452) [inline Features3/tests/features/addContact.feature]: Try out multiple combinations, like invalid email, name, phone, etc. Let's not repeat a single case twice.
````
|name|email|phone|contact-type|
|abc|abcgmail.com|9801234567|personal|
|abc|abcgmail.com|9801234567|professional|

Here, both of these cover a single case of invalid email. So no need to have two data, single one would be good enough.
```

Comment (kiranadh1452) [inline Features3/tests/features/addContact.feature]: Try out multiple combinations, like invalid email, name, phone, etc. Let's not repeat a single case twice.
````
|name|email|phone|contact-type|
|abc|abcgmail.com|9801234567|personal|
|abc|abcgmail.com|9801234567|professional|

Here, both of these cover a single case of invalid email. So no need to have two data, single one would be good enough.
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**